google/earthenterprise
# QT4 (EL8) Database Push Error Fix

Fixes the error dialog that occurs during a database push on EL8.

To test:

1. Install on EL8.
2. Build a database.
3. Push the database to the server.
4. Verify that no error dialog pops up, and that the progress dialog appears (may be very fast).
5. For bonus points: Slow down your localhost network connection and verify that the progress dialog works correctly (`tc qdisc add dev lo root handle 1:0 netem delay 500msec` will do the trick).
